application/core/services/chapter_service.py

```
@@ -244,19 +244,14 @@ def save_chapter_review(
             chapter.status = new_chapter_status
             self.chapter_repository.save(chapter)
 
-        # 使用数据库 repository
-        if self.chapter_review_repository:
-            return self.chapter_review_repository.upsert(
-                novel_id, chapter_number, status=status, memo=memo
+        # 使用数据库 repository；未装配时显式失败，不伪造返回值
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF003)